homeworks/7/chat-system/src/Client/Client.java

```diff
@@ -0,0 +1,74 @@
+package Client;
+
+import java.io.*;
+import java.net.Socket;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Client extends Thread {
+    private Socket socket = new Socket();
+    public Client(Socket socket){
+        this.socket = socket;
+    }
+
+    @Override
+    public void run(){
+        while (true) {
+            DataInputStream in = null;
+            try {
+                in = new DataInputStream(socket.getInputStream());
+                String fromServer = in.readUTF();
+                System.out.println(fromServer);
+            } catch (IOException e) {
+                try {
+                    if (in != null) {
+                        in.close();
+                    }
+                    socket.close();
+                } catch (IOException ex) {
+                    System.out.println(ex);
+                }
+            }
+        }
+    }
+
+    public static void main(String[] args) throws IOException {
+        Scanner scanner = new Scanner(System.in);
+        Socket socket = new Socket();
+        socket.setKeepAlive(true);
+        Client thread;
+        DataOutputStream dout = null;
+
+        while (true) {
+            String command = scanner.nextLine();
+            ArrayList<String> commandList = new ArrayList<>(List.of(command.split(" ")));
+            switch (commandList.get(0)) {
+                case "/connect":

```

Comment:
The ip and port are supposed to be formatted like this: "/connect [ip of chat server]:[port]". Not 
"[ip of chat server] [port]".